code/go/0chain.net/miner/server/grpc/handlers/get_client.go

```diff
@@ -0,0 +1,23 @@
+package handlers
+
+import (
+	"context"
+
+	"0chain.net/core/datastore"
+	minerproto "0chain.net/miner/proto/api/src/proto"
+	"google.golang.org/genproto/googleapis/api/httpbody"
+)
+
+func GetClient(ctx context.Context, req *minerproto.GetClientRequest) (*httpbody.HttpBody, error) {
+
+	response, err := datastore.GetEntityHandler(ctx, req.Id, datastore.GetEntityMetadata("client"))
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &httpbody.HttpBody{
+		// Done by datastore.ToJSONEntityReqResponse middelware. Need to find workaround.
```

Comment:
The [REST handler](https://github.com/dhruv0000/0chain/blob/6f79f7aaec63f4198434a32486b416f4ff65aa71/code/go/0chain.net/chaincore/client/handler.go#L16) supports `interface{}` type as an return value since [ToJSONResponse](https://github.com/dhruv0000/0chain/blob/6f79f7aaec63f4198434a32486b416f4ff65aa71/code/go/0chain.net/core/common/handler.go#L130) and [ToJSONReqResponse](https://github.com/dhruv0000/0chain/blob/6f79f7aaec63f4198434a32486b416f4ff65aa71/code/go/0chain.net/core/common/handler.go#L146) handles custom encoding for all the unknown return type.

//cc @kushthedude